src/code/z_vismono.c

```diff
@@ -1,10 +1,27 @@
+/**
+ * Color frame buffer effect to desaturate the colors.
+ */
```

Comment:
```suggestion
/**
 * Color frame buffer (cfb) effect to desaturate the colors.
 */
```
Just one small comment from me. The abbreviation `cfb` is used a lot in this file. It would be good to have this explicitly stated at the top of the file. Option if you want to do `CFB` instead

Reply:
I only used it in "cfbfrag" I think so I put that near the define where it first gets used